room/views.py

```diff
@@ -0,0 +1,65 @@
+from rest_framework_simplejwt.views import TokenObtainPairView
+from rest_framework.permissions import AllowAny
+from django.contrib.auth.models import User
+from django.contrib.auth import login
+from rest_framework import generics
+from django.http import HttpResponse
+from .models import Room
+from django.shortcuts import render, redirect
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import render
+
+from .models import Room, Message
```

Comment:
There are multiple same imports, also it should be in ascending way like this

from dotenv import load_dotenv
from celery import shared_task
from sp_api.api import Reports
from confluent_kafka import Producer
from sp_api.api import Notifications
from datetime import datetime, timedelta